test/suite/test_hs30.py

```diff
@@ -182,8 +182,8 @@ def test_insert_updates_hs(self):
         stat_cursor.close()
 
         if self.do_evict:
-            # Should have read nrows twice.
-            self.assertEqual(hs_read, nrows * 2)
+            # Should have atleast read nrows twice.
```

Comment:
space at least

Reply:
Updated in [c34c9c9](https://github.com/wiredtiger/wiredtiger/pull/7687/commits/c34c9c9bcd6eae0396bc5380495b1bb3d2a80d01)